app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,78 @@
+require './config/environment'
+
+class TweetsController < ApplicationController
+
+#All Tweets
+  get '/tweets' do
+    if logged_in?
+      @user = current_user
+      erb :'/tweets/tweets'
+    else
+      redirect to "/login"
+    end
+  end
+
+#New Tweet
+  get '/tweets/new' do
+    if logged_in?
+      erb :'/tweets/create_tweet'
+    else
+      redirect to "/login"
+    end
+  end
+
+#New Tweet- Form Submit
+  post '/tweets' do
+    if params[:content] == ""
+      redirect '/tweets/new'
+    else
+      @tweet = Tweet.create(:content => params["content"])
+      @tweet.user_id = current_user.id
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.create(content: params[:content], user: current_user)
```